aiida/tools/archive/implementations/sqlite/backend.py

```diff
@@ -202,7 +202,7 @@ def maintain(self, dry_run: bool = False, live: bool = True, **kwargs) -> None:
         raise NotImplementedError
 
     def get_info(self, statistics: bool = False, **kwargs) -> dict:
-        raise NotImplementedError
+        return {'Objects': len(list(self.list_objects()))}
```

Comment:
Is there a reason why this one doesn't use `super` as well? (also "objects" maybe in lowercase and with a nested dict to make it consistent with the others?)

```python
    def get_info(self, statistics: bool = False, **kwargs) -> dict:
        results = super().get_info(statistics=statistics)
        results['objects'] = {'count': len(list(self.list_objects()))}
        return results
```

Reply:
Fine for me to make it lower case, but `DiskObjectStoreRepositoryBackend.get_info` actually uses `Objects` with a capital.

Not using super was just an omission, will add it now.

Regarding the `get_orm_entities`, yes I kept the implementation, just changing the `verbose` variable to `statistics`.